pages/api/admin/users/readMany.ts

```diff
@@ -36,7 +36,7 @@ export default async (
                 },
               },
             }
-          : undefined),
+          : { status: UserStatus.Active }),
```

Comment:
is this to make sure that only active users are shown the user dashboard? It doesn't seem to be working correctly — maybe adding the status to the where section will allow for only active users to show up on the User Dashboard, we only want the admin to see inactive users in the Inactive Tab

Reply:
fixed !